### PR TITLE
PerfCounters enabled-by-default note added to 1.x packages

### PR DESCRIPTION
**Pull request by dvdstelt**
Solves #2884

### CODE REVIEW

File: nservicebus/operations/performance-counters_note_perfcounters_[1,).partial.md [hunk 1]
```diff
@@ -0,0 +1 @@
+While the standard performance counters in the NServiceBus 6 package are marked obsolete, for compatibility reasons they are still enabled by default.
```
Comment (SimonCropp) [inline nservicebus/operations/performance-counters_note_perfcounters_[1,).partial.md]: so this is true for NSB v7 as well?

Comment (dvdstelt) [reply nservicebus/operations/performance-counters_note_perfcounters_[1,).partial.md]: You mean the naming of the file isn't correct? In NSB7 they should be completely removed. How would I name the file, if it needs to be <2.0?

Comment (SimonCropp) [reply nservicebus/operations/performance-counters_note_perfcounters_[1,).partial.md]: sorry my comment was incorrect

**Comment by weralabaj** [comment]
LGTM, @Particular/metrics-maintainers feel free to merge on your own

**Pull request merged**